src/rhsm/connection.py

```diff
@@ -533,7 +510,7 @@ def _request(self, request_type, method, info=None):
         if not len(result['content']):
             return None
 
-        return json.loads(result['content'], object_hook=self._decode_dict)
+        return json.loads(result['content'])
```

Comment:
No, we don't need it anywhere except for after the read call (I just pushed a quick change; I missed one of the read calls, and re-reading to be able to better answer your question helped me catch it 👍 ). Unless I missed something, in my testing both json and simplejson will handle unicode properly in `loads` provided they are passed unicode in the first place. So, by moving the decode to the read call ([example](https://github.com/candlepin/python-rhsm/blob/fe0b5f4b2bf730fd9631ddc8a5e81eca0452fef8/src/rhsm/connection.py#L300)), we can knock it out early and avoid traversing the whole object graph.